source/doc-pages/guides/basics/repositories.md

```diff
@@ -0,0 +1,130 @@
+# Repositories
+
+In ROM repositories encapsulates access to domain-specific entities. They use
+relations and exposes a convenient interface for composing relations together
```

Comment:
“They” and “exposes” are not in agreement here (English, huh?).

Should be:

> They use relations and expose a convenient interface—

Or:

> They use relations to expose a convenient interface—